File: Dragon_Dungeons/Models/Bonus.cs
namespace Dragon_Dungeons.Models
{
  public class Bonus : RepoItem
  {
    public int Str { get; set; }
    public int Dex { get; set; }
    public int Con { get; set; }
    public int Intelligence { get; set; }
    public int Wis { get; set; }
    public int Cha { get; set; }
    public string CharacterId { get; set; }
  }
}

File: Dragon_Dungeons.client/src/models/Bonus.js
import { RepoItem } from "./RepoItem.js"

export class Bonus extends RepoItem {
  constructor(data) {
    super(data)
    this.str = data?.str
    this.dex = data?.dex
    this.con = data?.con
    this.int = data?.intelligence
    this.wis = data?.wis
    this.cha = data?.cha
    this.characterId = data?.characterId
  }
}